README.md

```diff
@@ -1 +1,188 @@
+- **Linters SetUp**
+- **Follow HTML and CSS best practice**
+- **Display Text**
+- **Follow GitHub flow**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://qusai122.github.io)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+
+<!--
+Example commands:
+```sh
+  cd my-folder
+  git clone git@github.com:KUBAHO3/Hello-Microverse.git
+```
+--->
+### Install
+Install this project with:
+npm
+Example command:
+```sh
+  cd my-project
+  npm init -y
+```
+### Usage
+To run the project, execute the following command:
+npm start or live server
+Example command:
+```sh
+  GitHub Pages Server
+```
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+### Deployment
+You can deploy this project using:
+GitHub Pages
+but it's not done yet
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **Qusai Issa**
+- GitHub: [@qusai122](https://github.com/qusai122)
+- LinkedIn: [Qusai Issa](https://www.linkedin.com/in/qusai-issa-ab5a59156/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## :telescope: Future Features <a name="future-features"></a>
+- [ ] **making it responsive**
+- [ ] **Adding Animation**
+- [ ] **Making it dynamic**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [https://github.com/qusai122/](../../issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## :star:️ Show your support <a name="support"></a>
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FAQ (optional) -->
+## :question: FAQ (OPTIONAL) <a name="faq"></a>
+- **[Question_1]**
+  - [Answer_1]
+- **[Question_2]**
+  - [Answer_2]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] The link to your LICENSE file leads to an error page. I believe this is because you don't have an MIT LICENSE file in your project. So please include an MIT license file in your project and ensure that the link to your license leads to the LICENSE and not to an error page. :+1: 
- [ ] You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

![image](https://user-images.githubusercontent.com/102699958/219438033-c46a89de-aa52-40ab-84b0-0ed363d4d508.png)